tools/cli/util.go

```diff
@@ -738,51 +738,71 @@ func newContextWithTimeout(c *cli.Context, timeout time.Duration) (context.Conte
 
 // process and validate input provided through cmd or file
 func processJSONInput(c *cli.Context) *commonpb.Payloads {
-	rawJson := processJSONInputHelper(c, jsonTypeInput)
-	if len(rawJson) == 0 {
-		return nil
-	}
+	jsonsRaw := readJSONInputs(c, jsonTypeInput)
+
+	var jsons []interface{}
+	for _, jsonRaw := range jsonsRaw {
+		if len(jsonRaw) == 0 {
```

Comment:
whats the scenario for this? passing an intentionally empty argument? Why would we not just ErrorAndExit in this case? Or is this an impossible scenario if invoked from the command line?

Reply:
How do we pass `nil`? `--input "null"` should work, right? If yes, I would validate this case and `ErrorAndExit("Empty input flag. Pass \"null\" for nil parameter value.")

Reply:
using `continue` was indeed not good.
Added support for passing `null`